src/components/common/ColonyHome/ColonyUnclaimedTransfers/ColonyUnclaimedTransfers.tsx

```diff
@@ -126,7 +117,7 @@ const ColonyUnclaimedTransfers = () => {
           <li>
             <Link
               className={styles.manageFundsLink}
-              to={`/colony/${name}/funds`}
+              to={`/colony/${colony?.name}/funds`}
```

Comment:
The `ActionButton` above is not visibly disabled when it's actually disabled. It could benefit from being grey out or similar, or hidden entirely.

It and the tooltip should also probably live in their own component (`ClaimButton`) (there's some duplication with `UnclaimedTransferItem`).